config/default.go

```diff
@@ -496,7 +496,9 @@ func OverwriteNeighborConfigWithPeerGroup(c *Neighbor, pg *PeerGroup) error {
 	overwriteConfig(&c.TtlSecurity.Config, &pg.TtlSecurity.Config, "neighbor.ttl-security.config", v)
 
 	if !v.IsSet("neighbor.afi-safis") {
-		c.AfiSafis = pg.AfiSafis
+		for _, afiSafi := range pg.AfiSafis {
+			c.AfiSafis = append(c.AfiSafis, afiSafi)
+		}
```

Comment:
Each peer has own slice but still shares State object so one peer modifies a State object then it affects the rest of peers?

Reply:
yes, an example is

Reply:
AfiSafi.State is not a pointer but a value and modification to one won't affect others.

Reply:
oops. pushed, thanks.